programs/vote/src/vote_instruction.rs

```diff
@@ -49,6 +49,9 @@ pub enum VoteInstruction {
     ///  to withdraw
     Authorize(Pubkey, VoteAuthorize),
 
+    /// Update the vote account's node id
+    UpdateNode(Pubkey),
```

Comment:
This is a good example of how we're very unprepared for locking down the cluster binary interface -- shuffling the `Vote` and `Withdraw` instructions up by 1 would be a hard fork.   We have no tests for any of this 😱